assets/components/organisms/media/media.scss

```diff
@@ -27,3 +27,30 @@ figcaption {
     }
   }
 }
+
+.media-map {
+  display: flex;
+  flex: 1 1 100%;
+  position: relative;
+
+  figure {
+    flex: 1 1 100%;
+  }
+
+  picture {
+    width: 100%;
+    margin-bottom: 0;
+  }
+
+  img {
+    height: 100%;
+    object-fit: cover;
```

Comment:
Because we must support IE11+, can you add a [polyfill](https://github.com/bfred-it/object-fit-images) and document how to use it 😉